lms/djangoapps/mobile_api/groups/views.py

```diff
@@ -249,104 +64,93 @@ class GroupsInvite(generics.CreateAPIView):
 
         POST /groups/invite/<group-id>/members
 
+        Parameters: members : int,int,int... 
+
 
     **Response Values**
 
-        {"success": "true"}
+        {"success" : "true"}        If adding all the member succeeded.
+        
+        {"error" : "error message", 
+         "member" : "id"}           If one of the members provided can't be added to the group. 
+                                    The error message pertains to the first member that couldn't be added.
+                                    Note that either all the memebers are added or none at all. 
     """
     authentication_classes = (OAuth2Authentication, SessionAuthentication)
     permission_classes = (permissions.IsAuthenticated,)
     
     def create(self, request, *args, **kwargs):
-        member_ids = request.POST['member-ids']
-        oauth_token = request.POST['oauth-token']
-        return Response(
-            {"success": "true"}
-        )
-
-
-class GroupsMembers(generics.RetrieveAPIView): 
+        graph = facebook.GraphAPI(facebook.get_app_access_token(_APP_ID, _APP_SECRET))
```

Comment:
There seems to be a lot of validation logic in here that should be abstracted out to Rest-Framework's serializers. They are the location to do all of the validation logic --- and they provide alot of the basic validation for you out of the box. http://tomchristie.github.io/rest-framework-2-docs/api-guide/serializers

i imagine you will want to use custom fields.

Let me know if you need some guidance setting this up, they can be tricky to set up and dive into without some background.